test/e2e/e2e_local_queue_defaulting_test.go

```diff
@@ -76,6 +77,44 @@ var _ = Describe("LocalQueueDefaulting", Label("local-queue-default"), Ordered,
 			deleteNamespace(ctx, ns)
 		})
 
+		JustAfterEach(func(ctx context.Context) {
```

Comment:
Could you refactor this as a utility func and reference in all the places that you use it?